github-extension/background.js

```diff
@@ -0,0 +1,113 @@
+const API_KEY = "INSERT_API_KEY_HERE";
+const BASE_URL = "https://api.github.com/graphql";
+
+// 15 minutes
+const UPDATE_INTERVAL = 15 * 60 * 1000;
+
+let lastUpdated;
+let cards;
+
+const requestBody = `{"query": "query { viewer { login contributedRepositories(first: 5, affiliations:[OWNER,COLLABORATOR], orderBy: {field: UPDATED_AT, direction: DESC}) { edges { node { nameWithOwner issues(first: 30, states: OPEN, orderBy: {field: UPDATED_AT, direction: DESC}) { edges { node { number title url updatedAt author { login avatarUrl } authorAssociation assignees(last:10) { edges { node { login } } } } } } pullRequests(first:30, states:OPEN, orderBy:{field: UPDATED_AT, direction: DESC}) { edges { node { number title url updatedAt author { login avatarUrl } assignees(last: 10) { edges { node { login } } } reviewRequests(last:5) { edges { node { reviewer { login } } } } } } } } } } } }"}`;
+
+const getCards = () =>
+  new Promise((resolve, reject) => {
```

Comment:
You're not actually rejecting under any condition here, so you can either just pass resolve, or add a reject case